cantabular/table.go

```diff
@@ -51,7 +51,7 @@ func (c *Client) ParseTable(table Table) (*bufio.Reader, error) {
 
 	// Obtain the CSV rows according to the cantabular dimensions and counts
 	for i, count := range table.Values {
-		row := createCSVRow(table.Dimensions, i, count)
+		row := createCSVRow(table.Dimensions, i, int(count))
```

Comment:
Kind of seems pointless to handle decimals if we are going to convert to int for the CSV?

Reply:
Yep. Not sure how I missed that